File: PHP/Driving Institute exercise/Institute/Plan.php
<?php

class Plan {
    private string $name;
    private float $price;
    private string $services;
    private int $duration; 
    private float $totalPrice;
    
    public function __construct(string $name, int $price,int $duration, string $services){
        $this->name = $name;
        $this->price = $price;
        $this->duration = $duration; 
        $this->services = $services;
        $this->totalPrice = $this->price * $this->duration;
    }

    public function getName(): string {
        return $this->name;
    }

    public function getPrice():float{
        return $this->price;
    }

    public function getDuration():int{
        return $this->duration;
    }

    public function getServices(): string {
        return $this->services;
    }

    public function getTotalPrice(): float {
        return $this->totalPrice;
    }

 
    
}

File: JAVA/Driving Institute exercise/com/Institute/Plan.java
package com.Institute;

public class Plan{
    private String name;
    private double price;
    private String services;
    private int duration;
    private double totalPrice;


    public Plan (String name, double price, int duration, String services) {
        this.name = name;
        this.price = price;
        this.services = services;
        this.duration=duration; 
        this.totalPrice= totalPrice;
   }

    public String getName() {
        return this.name;
    }

    public double getPrice() {
        return this.price;
    }

    public String getServices() {
        return this.services;
    }
    
    public int getDuration(){
        return this.duration;
    }
    
    public double getTotalPrice(){
        return this.totalPrice= this.getPrice()*this.getDuration();
    }

   
    

}